### ISSUE TITLE
Calling forceUpdate twice in succession doesn't force a re-render

### ISSUE THREAD

**Issue by latviancoder** [CharlesStover/use-force-update#9]
Here is codesandbox with example:

https://codesandbox.io/s/tender-noether-cnvrz

I assume this happens because React batches two state updates and because in the end state stays the same (true -> false -> true) rerender doesn't happen at all.

**Comment by ramblehead** [comment]
```js
const [, doUpdate] = useState(now());
const update = useCallback((): void => doUpdate(now()), [doUpdate]);

// ...
// Call update() as required
```

```now()``` could be one of the following:
```const now = () => performance.now();```
or
```import now from '```[performance-now](https://github.com/myrne/performance-now)```';```

The second variant is useful in SSR.

**Comment by latviancoder** [comment]
Or you could simply increment a number the way @sophiebits suggests here:

https://mobile.twitter.com/sophiebits/status/1141935418893463553

**Comment by CharlesStover** [comment]
The number increment was the original implementation. Sophie would be a great citation for that.

**Comment by CharlesStover** [comment]
Thank you for addressing this issue. This will be fixed in `1.0.6` today. :)
